### DISCUSSION TITLE
need Video Guide For  AD

### DISCUSSION THREAD

**Discussion by landtools** [Stephan-S/FS19_AutoDrive#2152] [General]
good morning everyone, after so many years playing with courseplay, i want to try this too. is it possible to have a guide for beginners ??

**Comment by yann-mrt** [comment]
Good day,
Their is a wiki page that explains the main function here - https://github.com/Stephan-S/FS19_AutoDrive/wiki -. But the best for me to learn was to go to YouTube and look for tuto. Depend the language you can understand. Iwan (Iwan1803 have a nice one in German that you can make translation in YouTube with integrated close caption).
As well look for discord channel for fast help, AD have a good one.
